packages/webapp/src/views/land/legal-party-add-start.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'form-layout.html' %}
+
+{% set showSaveLink = true %}
```

Comment:
Just noticed this, might have missed a few, but showSaveLink is deprecated since defraID integration and should be removed.

Reply:
done